test/test.css

```diff
@@ -0,0 +1,47 @@
+ body {
```

Comment:
does the test need css to work? i guess maybe only `font-family` is important

Reply:
heh no, not really, I was just looking at it from the browser and needed some styling, I'll remove that file

Reply:
ya i removed most of the dom in the last PR since it's for test only :ambulance: